po/common/fr.po

```diff
@@ -4607,20 +4607,69 @@ msgstr "Pas ou peu d'additifs"
 
 msgctxt "preference_not_important"
 msgid "Not important"
-msgstr ""
+msgstr "Pas important"
 
 msgctxt "preference_important"
 msgid "Important"
-msgstr ""
+msgstr "Important"
 
 msgctxt "preference_very_important"
 msgid "Very important"
-msgstr ""
+msgstr "Très important"
 
 msgctxt "preference_mandatory"
 msgid "Mandatory"
 msgstr "Obligatoire"
 
+msgctxt "packaging_alt"
+msgid "Recycling instructions and/or packaging information"
+msgstr "Instruction de recyclage et/ou information d'emballage"
+
+msgctxt "image_packaging"
+msgid "Recycling instructions and/or packaging information picture"
+msgstr "Image d'instructions de recyclage et/ou information d'emballage"
+
+msgctxt "image_packaging_url"
+msgid "Link to recycling instructions and/or packaging information photo"
+msgstr "Lien vers la photo des instructions de recyclage et/ou d'information d'emballage"
+
+# Do not translate the file name
+msgctxt "import_photos_format_packaging"
+msgid "3001234567890.packaging_es.jpg: recycling instructions in Spanish."
+msgstr "3001234567890.packaging_fr.jpg: instructions de recyclage en français."
```

Comment:
Is this correct? it says "Do not translate the file name"

Reply:
I translated it correctly ;)

Reply:
fair enough. :D